apps/admin-gui/src/app/shared/components/create-service-member-dialog/create-service-member-dialog.component.html

```diff
@@ -77,7 +77,7 @@ <h1 mat-dialog-title>{{'DIALOGS.CREATE_SERVICE_MEMBER.TITLE' | translate}}</h1>
         <h6>{{'DIALOGS.CREATE_SERVICE_MEMBER.ASSOCIATED_USERS' | translate}}</h6>
         <mat-list>
           <mat-list-item *ngFor="let member of assignedMembers">
-            <p class="truncate center-content">
+            <p class="truncate center-content d-flex align-items-center">
```

Comment:
Please add also something like class="me-0" to the search icon in the third step to center the icon:
![image](https://user-images.githubusercontent.com/70371638/220957892-0cf41325-367e-450e-8c5b-fc3082383348.png) 
![image](https://user-images.githubusercontent.com/70371638/220958154-181e6b67-2b42-4d10-9218-4af3d7535e96.png)

Reply:
Made the changes to styles.scss instead of _styles.scss, so they weren't included in the commit...